weave/server.py

```diff
@@ -238,8 +238,14 @@ def capture_weave_server_logs(log_level: int = logging.INFO):
     root_logger = logging.getLogger()
     root_logger.setLevel(log_level)
 
+    console_log_settings: typing.Optional[logs.LogSettings] = None
```

Comment:
This is a great patch for immediate fix, thank you! I am worried that we will also get erroneous errors in non-notebook situation. Greping for `logging.error(`, I think there are 2 cases (vectorize.py and projection_utils.py) that probably should be warnings instead. As they might clutter the logs of non-notebook execution as well.

Reply:
updated them to warnings